### ISSUE TITLE
Google Analytics tracking code 

### ISSUE THREAD

**Issue by emilyhobson** [stamen/bluegreenway#179]
Need GA ID from Justin
Implement

**Comment by ericsoco** [comment]
Needs to track on hash change, like Atlas:
https://github.com/stamen/atlas-of-emotions/commit/f0bca0ff554c1a738849fca193fc497fe4d5fbbd

**Comment by emilyhobson** [comment]
Hi Evan,

I looked through my files and I found the info for the analytics on the SFPA site. That’s here:

https://analytics.google.com/analytics/web
login: feedback@sfparksalliance.org
password:

https://onetimesecret.com/secret/besn04zbrsvpjp2d3fa71ibsev5rien
(only works one time, expires in 7 days)

The login email is actually a legacy account from before SFPA had an organizational account.

If you want me to set up a BGW analytics code let me know. If one of you uses the secret link for the password I can send more links for the other folks. (Since the link can only be used once.)

Thanks
Justin
